apps/src/ReduxCDO.js

```diff
@@ -0,0 +1,34 @@
+/** @file Code.org configured store-creation method.
+ *  @see http://redux.js.org/docs/api/createStore.html */
+// Strict linting: Absorb into global config when possible
+/* jshint
+ unused: true,
+ eqeqeq: true,
+ maxlen: 120
+ */
+'use strict';
+
+var redux = require('redux');
+
+/**
+ * Creates a store configured for use the way we want for Code.org.
+ * @see http://redux.js.org/docs/api/createStore.html
+ * @param {!function} reducer
+ * @param {?} [initialState] optionally give the store an initial state.
+ * @return {Store} Configured Redux store, ready for use.
+ */
+module.exports = function createStoreCDO(reducer, initialState) {
+  if (process.env.NODE_ENV !== "production") {
+    var createLogger = require('redux-logger');
```

Comment:
Oh, that makes sense. Could we not still have it within a conditional at the top of the file?